libs/designer/src/lib/core/templates/utils/helper.ts

```diff
@@ -84,6 +111,15 @@ export const getFilteredTemplates = (
     });
     return hasDetailFilters;
   });
+
+  if (!filters.keyword) {
```

Comment:
Nit: Might be easier to return this at the beginning instead of waiting for all the logic to be executed

Reply:
We are returning the executed value which is filteredTemplateEntries (declared in line 89). This function is running two logics: filters and search (from keyword). If there's no keyword, we will still run through the filter logic, just won't go through the search logic below this line.

Reply:
ohh ok ok, makes sense. Sorry I got confused. Sounds good then!